common.lic

```diff
@@ -268,9 +268,10 @@ module DRC
     end
   end
 
-  def hide?
+  def hide?(*hide_type)
+    hide_type.any? ? hide_type = hide_type[0] : hide_type = 'hide' 
```

Comment:
We can replace both of these lines with 
```
def hide?(hide_type = 'hide')
```

and add hide_type to `base.yaml` and `base-empty.yaml` with the default value.